pkg/git/repository_test.go

```diff
@@ -0,0 +1,149 @@
+package git
+
+import (
+	"testing"
+	"time"
+)
+
+func TestGetRootDir(t *testing.T) {
```

Comment:
@mfojtik @deads2k do we run unit tests against library-go?

Reply:
@mfojtik @deads2k bump

Reply:
Yes we do, in Travis